best-practices/styling/index.html

```diff
@@ -1016,17 +1054,180 @@ <h3>Wrapping and breaking text</h3>
 			<section id="page-breaking">
 				<h3>Page breaking and column layout</h3>
 
-				<div class="issue" title="Seeking an example of forcing page breaks"><span/></div>
-
-				<div class="issue" data-number="93" title="Seeking an example of avoiding page breaks within
+				<div class="issue" title="Need HTML and CSS of forcing page breaks"><span/></div>
+				<aside class="example" title="Forced page break between front matter and body matter"></aside>
+					<p>In order to obtain the following desired layout for the above HTML code,</p>
+					<pre><code class="formatted-braille">⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠁⠰⠊⠧
+⠠⠊⠝⠙⠕⠝⠑⠎⠊⠁⠀⠐⠐⠐⠐⠐⠐⠐⠐⠐⠐⠐⠐⠐⠐⠐⠐⠐⠐⠐⠐⠐⠐⠐⠀⠼⠛⠚
+⠠⠊⠗⠁⠝⠀⠐⠐⠐⠐⠐⠐⠐⠐⠐⠐⠐⠐⠐⠐⠐⠐⠐⠐⠐⠐⠐⠐⠐⠐⠐⠐⠐⠐⠀⠼⠛⠁
+⠠⠊⠗⠑⠇⠁⠝⠙⠀⠐⠐⠐⠐⠐⠐⠐⠐⠐⠐⠐⠐⠐⠐⠐⠐⠐⠐⠐⠐⠐⠐⠐⠐⠐⠀⠼⠛⠉
+⠠⠊⠎⠗⠁⠑⠇⠀⠐⠐⠐⠐⠐⠐⠐⠐⠐⠐⠐⠐⠐⠐⠐⠐⠐⠐⠐⠐⠐⠐⠐⠐⠐⠐⠀⠼⠛⠙
+⠠⠊⠞⠁⠇⠽⠀⠐⠐⠐⠐⠐⠐⠐⠐⠐⠐⠐⠐⠐⠐⠐⠐⠐⠐⠐⠐⠐⠐⠐⠐⠐⠐⠐⠀⠼⠛⠋
```

Comment:
Could we change this TOC into something else, e.g. a simple list?

> - Indonesia
> - Iran
> - Ireland
> - Israel
> - Italy

I want as little distraction from what we are trying to show (the `page-break-after` property). (A [table of contents with guide dots](daisy.github.io/ebraille/best-practices/styling/#example-table-of-contents-with-guide-dots) requires a lot of CSS, so a lot of distraction.) Could we also drop the headers/footers with page numbers?

Reply:
@wfree-aph I've been looking at the rest of the example, trying to figure out the HTML, but I need some help with it.

Or perhaps we just need a simplified example, it seems that for this example there will be quite a lot of different markup and styling. We could possibly just drop everything after the title "World Braille Usage Third Edition Braille Usage by Country".

(By the way, I would even drop the "(cont.)", as this is something related to volume breaking, something we haven't touched yet, as we're focusing on electronic reading systems now, and will explore embossing only later).

Reply:
@bertfrees Sorry for my delay. A table of contents with guide dots is the requirements of a region of braille formatting. I don't know that we can just drop it entirely. You could leave it as a noted limitation and try to figure it out later though.

Some regions will make a simple list, so that will make sense. It will show how to make the list and include the links to those page numbers.

Reply:
The point is that this example is not about a TOC, it's about making the bodymatter start on a new page.

I don't want to mess with the braille formatting requirements, I was thinking we would just change the markup so that it wouldn't be a TOC anymore, but something else with simpler CSS.

The other solution is to omit any CSS that is not relevant to the feature that the example is actually about. It would be a departure from how it was done until now in the rest of the document, but it could work.

Reply:
Ah. My fault for misunderstanding. In that case, making it a simple list works entirely. Just remove the guide dots and page numbers. At that point it's just a list of country names but takes up the same amount of space.

Reply:
@wfree Can you help me with the markup of the first elements in the bodymatter? No need to give me HTML code. Just tell me what these element are:

1. ```
          ,WORLD ,BRAILLE ,USAGE
               ,THIRD ,EDITION           
   ```
2. ```
   ,BRAILLE ,USAGE BY ,COUNTRY "<CONT4"> 
   ```
3. ```                                      
                   ,ECUADOR              
   ```
4. ```
   ,LANGUAGE3 ,SPANISH "<USUALLY UNCONTRACTED">                      
   ```
5. ```
    ,BRAILLE ,STANDARDS ,SET BY3
   ```      
6. (I assume this is a list)
   ```
   ,LATIN ,AMERICAN ,BRAILLE ,COUNCIL    
     "<,COMISI^/ON ,BRAILLE              
     ,LATINOAMERICANA.> "<,,CBL">        
   ,LATIN ,AMERICAN ,BLIND ,UNION        
     "<,UNI^/ON ,LATINOAMERICANA DE      
     CIEGOS"> "<,,ULAC">                 
   WWW4ULACDIGITAL4ORG                   
   ,BRAILLE@AULACDIGITAL4ORG             
   ```

Reply:
1 through 3 are all separate h1s (centered headings). 4 is a solitary list item. 5 is an h2 (cell-5 heading). And 6 is a list as you assumed.

Reply:
Thanks. I ended up keeping the TOC and most of the content on the next page. In the CSS I added this comment:

```css
/* For clarity, any CSS that is not relevant for the page break was left out */
```